CMakeLists.txt

```diff
@@ -44,15 +44,20 @@ install(DIRECTORY ${CATKIN_DEVEL_PREFIX}/lib/
                        PATTERN "libumfpack*"
 )
 cs_export(INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include/suitesparse
-          LIBRARIES amd
+          LIBRARIES
+                    cholmod
+                    amd
                     btf
                     camd
                     colamd
                     ccolamd
-                    cholmod
                     cxsparse
                     klu
                     ldl
                     rbio
                     spqr
-                    umfpack)
+                    suitesparseconfig
+                    umfpack
+                    lapack
```

Comment:
can you order these alphabetically?

Reply:
I could a little more but not entirely because of the interdependence. For instance `cholmod` must be above most of them including `amd`. And it is a little tedious to figure the requirements out. This order works so I'd prefer to keep it.

Reply:
Interesting, I didn't know that the order matters here :) ok for me if you keep it